baseplate/lib/metrics.py

```diff
@@ -302,6 +303,9 @@ def send(self, elapsed: float) -> None:
 
         """
         serialized = self.name + (f":{(elapsed * 1000.0):g}|ms".encode())
+        if self.sample_rate != 1.0:
```

Comment:
I don't know how python handle float numbers, but the general rule of thumb is that don't use `==` and `!=` on float numbers. In this case I'd suggest to use `self.sample_rate < 1.0` instead.

Reply:
👍